internal/engine/eval/trusty/actions.go

```diff
@@ -324,7 +324,7 @@ func (sph *summaryPrHandler) generateSummary() (string, error) {
 			// (2) we don't suggest malicious packages, I
 			// suggest getting rid of this check
 			// altogether.
-			if altData.Score != nil && *altData.Score <= lowScorePackages[alternative.Dependency.Name].Score {
+			if altData.Score != nil && *altData.Score != 0 && *altData.Score <= lowScorePackages[alternative.Dependency.Name].Score {
```

Comment:
The diff here is a bit confusing: this only adds `*altData.Score != 0` which fixes bug #4945

Reply:
should this be moved to a small and dedicated function with an appropriate explanation? you're right that it's a bit confusing and having some docs in the code would probably help.